<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientGroupBox</class>
 <widget class="QGroupBox" name="PatientGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="title">
   <string notr="true">Patient Information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="allscriptsTitle">
       <property name="toolTip">
        <string notr="true">MR Number</string>
       </property>
       <property name="text">
        <string notr="true">MR Number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="allscriptsLabel">
       <property name="toolTip">
        <string notr="true">MR Number</string>
       </property>
       <property name="text">
        <string notr="true">PH ALLSCRIPTS ID</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lastTitle">
       <property name="toolTip">
        <string notr="true">Last name</string>
       </property>
       <property name="text">
        <string notr="true">Last name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lastLabel">
       <property name="toolTip">
        <string notr="true">Last name</string>
       </property>
       <property name="text">
        <string notr="true">PH LAST</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="firstTitle">
       <property name="toolTip">
        <string notr="true">First name</string>
       </property>
       <property name="text">
        <string notr="true">First name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="firstLabel">
       <property name="toolTip">
        <string notr="true">First name</string>
       </property>
       <property name="text">
        <string notr="true">PH FIRST</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="dobTitle">
       <property name="toolTip">
        <string notr="true">Date of birth</string>
       </property>
       <property name="text">
        <string notr="true">DOB:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dobLabel">
       <property name="toolTip">
        <string notr="true">Date of birth</string>
       </property>
       <property name="text">
        <string notr="true">PH DOB</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="activeTitle">
       <property name="toolTip">
        <string notr="true">Active or inactive status</string>
       </property>
       <property name="text">
        <string notr="true">Active:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="activeLabel">
       <property name="toolTip">
        <string notr="true">Active or inactive status</string>
       </property>
       <property name="text">
        <string notr="true">PH ACTIVE</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
